lib_stusb/STM32_USB_Device_Library/Class/CCID/src/usbd_ccid_if.c

```diff
@@ -614,11 +621,12 @@ void io_usb_ccid_set_card_inserted(unsigned int inserted) {
 #endif // HAVE_CCID_INTERRUPT
 }
 
+void io_usb_ccid_configure_pinpad(uint8_t enabled) {
+    volatile uint8_t *cfgDesc = NULL;
 
-
-
-
-
+    cfgDesc = (volatile uint8_t *)USBD_GetPinPadOffset();

```

Comment:
`const volatile uint8_t *cfgDesc = USBD_GetPinPadOffset();`
this should work no?
And later cast to `void *`  for `nvm_write` call.

Reply:
Done